src/renderers/shaders/ShaderChunk/bsdfs.glsl

```diff
@@ -28,7 +28,7 @@ vec3 F_Schlick( const in vec3 specularColor, const in float dotLH ) {
 	// Optimized variant (presented by Epic at SIGGRAPH '13)
 	float fresnel = exp2( ( -5.55437 * dotLH - 6.98316 ) * dotLH );
 
-	return ( 1.0 - specularColor ) * fresnel + specularColor;
+	return mix( specularColor, grazingColor, fresnel );
```

Comment:
Remember that F_Schlick is designed as a dielectric Fresnel function that is not valid for metal objects that exhibit conductor Fresnel reflection.  Conductor Fresnel does not necessarily go to white at grazing angles, rather it a specific color defined by the properties of the metal -- a yellowish color for things like gold.

We could calculate a separate conductor Fresnel and blend between it and the F_Schlick dieletric Fresnel?  I think that a fast conductor Fresnel approximation is likely nearly what I created.  The only real enhancement one could make would be to allow for direct specification of the grazingColor as it does differ a bit from the F0 color.